tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
-  "strict": true,
   "compilerOptions": {
-    "target": "esnext",
+    "target": "ES2018",
     "strict": true,
     "esModuleInterop": true,
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
+    "noEmit": true,
```

Comment:
Since this package is using `tsup` to emit the compiled JS files, this option is handy so files aren't emitted when attempting to typecheck using `tsc`.